src/pages/index.js

```diff
@@ -34,6 +34,7 @@ export default function Home() {
       />
       <br />
       new form --{">"} <Link href="/form">Click me</Link>
+      <Link href="/comic_pricing"> new new feature</Link>
```

Comment:
```suggestion
      <Link href="/comic_pricing">new new feature</Link>
```

Use `Stack` if you want some space.